linux_docker_resources/Dockerfile

```diff
@@ -109,7 +109,7 @@ RUN if test ${UBUNTU_DISTRO} != xenial; then apt-get update && apt-get install -
 RUN if test ${UBUNTU_DISTRO} != xenial; then apt-get update && apt-get install --no-install-recommends -y libpoco-dev; fi
 
 # Install build dependencies for rviz et al.
-RUN apt-get update && apt-get install --no-install-recommends -y libassimp-dev libcurl4-openssl-dev libfreetype6-dev libgles2-mesa-dev libglu1-mesa libqt5core5a libqt5gui5 libqt5opengl5 libqt5widgets5 libxaw7-dev libxrandr-dev qtbase5-dev
+RUN apt-get update && apt-get install --no-install-recommends -y libassimp-dev libcurl4-openssl-dev libfreetype6-dev libgles2-mesa-dev libglu1-mesa-dev libqt5core5a libqt5gui5 libqt5opengl5 libqt5widgets5 libxaw7-dev libxrandr-dev qtbase5-dev
```

Comment:
How is this change related to `armhf` support?

Reply:
Before [![Build Status](https://ci.ros2.org/buildStatus/icon?job=test_ci_linux-armhf&build=4)](https://ci.ros2.org/view/All/job/test_ci_linux-armhf/4/consoleFull#console-section-299)

After [![Build Status](https://ci.ros2.org/buildStatus/icon?job=test_ci_linux-armhf&build=5)](https://ci.ros2.org/job/test_ci_linux-armhf/5/)

I haven't taken the time to figure out what's pulling in the deb package or providing that header for our other platforms.

Reply:
```
07:49:55 In file included from /home/jenkins-agent/workspace/test_ci_linux-armhf/ws/build/rviz_ogre_vendor/ogre-master-ca665a6-prefix/src/ogre-master-ca665a6/RenderSystems/GL/include/OgreGLPrerequisites.h:63:0,
07:49:55                  from /home/jenkins-agent/workspace/test_ci_linux-armhf/ws/build/rviz_ogre_vendor/ogre-master-ca665a6-prefix/src/ogre-master-ca665a6/RenderSystems/GL/src/GLSL/include/OgreGLSLExtSupport.h:34,
07:49:55                  from /home/jenkins-agent/workspace/test_ci_linux-armhf/ws/build/rviz_ogre_vendor/ogre-master-ca665a6-prefix/src/ogre-master-ca665a6/RenderSystems/GL/src/GLSL/src/OgreGLSLExtSupport.cpp:31:
07:49:55 /home/jenkins-agent/workspace/test_ci_linux-armhf/ws/build/rviz_ogre_vendor/ogre-master-ca665a6-prefix/src/ogre-master-ca665a6/RenderSystems/GL/include/GL/glew.h:1180:14: fatal error: GL/glu.h: No such file or directory
07:49:55  #    include <GL/glu.h>
07:49:55               ^~~~~~~~~~
```

Reply:
1. [ogre_add_library_to_folder](https://github.com/OGRECave/ogre/blob/ca665a69a5ca47608ee0ad304b2596ff866616da/CMake/Utils/OgreAddTargets.cmake#L105)
1. [CMakeLists.txt](https://github.com/OGRECave/ogre/blob/ca665a69a5ca47608ee0ad304b2596ff866616da/RenderSystems/GL/CMakeLists.txt#L71)
1. [OgreGLSLExtSupport.cpp:31](https://github.com/OGRECave/ogre/blob/ca665a69a5ca47608ee0ad304b2596ff866616da/RenderSystems/GL/src/GLSL/src/OgreGLSLExtSupport.cpp#L31)
1. [OgreGLSLExtSupport.h:34](https://github.com/OGRECave/ogre/blob/ca665a69a5ca47608ee0ad304b2596ff866616da/RenderSystems/GL/src/GLSL/include/OgreGLSLExtSupport.h#L34)
1. [OgreGLPrerequisites.h:63](https://github.com/OGRECave/ogre/blob/ca665a69a5ca47608ee0ad304b2596ff866616da/RenderSystems/GL/include/OgreGLPrerequisites.h#L62-L64)
1. [GL/glew.h:1180](https://github.com/OGRECave/ogre/blob/ca665a69a5ca47608ee0ad304b2596ff866616da/RenderSystems/GL/include/GL/glew.h#L1175-L1182)

:man_shrugging:

Reply:
It should have been pulled in as a dependency of `qtbase5-dev` or `libsdl1.2-dev`. From a Bionic ARM instance:
```
$ apt-cache rdepends libglu1-mesa-dev
...
 |libsdl1.2-dev
...
 |qtbase5-dev
...
```

...the logs show both of those being installed, but the pipe indicates there are other options to satisfy that dependency. But the other option (libglu-dev) is a virtual package provided by libglu1-mesa-dev, so I have no idea how they were installed without libglu1-mesa-dev present as well.